IM_calculation/IM/intensity_measures.py

```diff
@@ -67,8 +67,11 @@ def calc_rotd(
 
     thetas = np.deg2rad(np.arange(min_angle, max_angle, delta_theta))
     rotation_matrices = np.asarray([np.cos(thetas), np.sin(thetas)])
+    rotds = []
```

Comment:
Create zeros and edit line by line rather than using python lists as a placeholder.

Otherwise looks good.